docs/sources/set-up/migrate/deploy-alloy-with-otel-collector.md

```diff
@@ -0,0 +1,30 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/set-up/migrate/deploy-alloy-with-otel-collector/
+description: Learn how to deploy Grafana Alloy alongside another distribution of the OpenTelemetry Collector
```

Comment:
Are Alloy and the other distribution meant to ingest the same telemetry? Or do they complement each other? If they ingest the same telemetry, then the users would need to set up some way for that telemetry to be funnelled to both collectors? I'd expect this to be part of this doc, but the solution to this problem really depends on what sort of receiver is being used.